openapi-cli/src/test/java/io/ballerina/openapi/CodeGeneratorTest.java

```diff
@@ -456,39 +456,39 @@ public void generateSkeletonForOperationFilter() {
         }
     }
 
-    @Test(description = "Test code generation when no schemas given")
-    public void testCodeGenerationWithoutSchemas() {
-        String definitionPath = RES_DIR.resolve("no_schema.yaml").toString();
-        BallerinaCodeGenerator generator = new BallerinaCodeGenerator();
-        try {
-            generator.generateClient(definitionPath, resourcePath.toString(), filter, false, false);
-            boolean hasTypeFileGenerated = Files.exists(resourcePath.resolve("client.bal")) &&
-                    Files.notExists(resourcePath.resolve("types.bal"));
-            Assert.assertTrue(hasTypeFileGenerated, "Empty types.bal file has been generated");
-
-        } catch (IOException | BallerinaOpenApiException | FormatterException e) {
-            Assert.fail("Error while generating the client. " + e.getMessage());
-        } finally {
-            deleteGeneratedFiles("client.bal");
-        }
-    }
-
-    @Test(description = "Test code generation when no schemas given in service")
-    public void testCodeGenerationWithoutSchemasService() {
-        final String serviceName = "no_schema";
-        String definitionPath = RES_DIR.resolve("no_schema.yaml").toString();
-        BallerinaCodeGenerator generator = new BallerinaCodeGenerator();
-        try {
-            generator.generateService(definitionPath, serviceName, resourcePath.toString(),  filter, false);
-            boolean hasTypeFileGenerated = Files.exists(resourcePath.resolve("no_schema_service.bal")) &&
-                    Files.notExists(resourcePath.resolve("types.bal"));
-            Assert.assertTrue(hasTypeFileGenerated, "Empty types.bal file has been generated");
-        } catch (IOException | FormatterException | BallerinaOpenApiException e) {
-            Assert.fail("Error while generating the service. " + e.getMessage());
-        } finally {
-            deleteGeneratedFiles("no_schema_service.bal");
-        }
-    }
+//    @Test(description = "Test code generation when no schemas given")
+//    public void testCodeGenerationWithoutSchemas() {
+//        String definitionPath = RES_DIR.resolve("no_schema.yaml").toString();
+//        BallerinaCodeGenerator generator = new BallerinaCodeGenerator();
+//        try {
+//            generator.generateClient(definitionPath, resourcePath.toString(), filter, false, false);
+//            boolean hasTypeFileGenerated = Files.exists(resourcePath.resolve("client.bal")) &&
+//                    Files.notExists(resourcePath.resolve("types.bal"));
+//            Assert.assertTrue(hasTypeFileGenerated, "Empty types.bal file has been generated");
+//
+//        } catch (IOException | BallerinaOpenApiException | FormatterException e) {
+//            Assert.fail("Error while generating the client. " + e.getMessage());
+//        } finally {
+//            deleteGeneratedFiles("client.bal");
```

Comment:
Shall we either enable or remove these commented test cases?

Reply:
Removed